relay-metrics/src/aggregation.rs

```diff
@@ -2615,56 +2606,85 @@ mod tests {
         "###);
     }
 
+    #[test]
+    fn test_capped_iter_empty() {
+        let buckets = vec![];
+
+        let mut iter = CappedBucketIter::new(buckets.into_iter(), 200);
+        assert!(iter.next().is_none());
+    }
+
     #[test]
     fn test_aggregator_cost_tracking() {
```

Comment:
im using `breakdowns` variant here as it allows custom names, which is required here to make the bucketkeys different from each other